pkg/rbd/nodeserver.go

```diff
@@ -302,7 +302,30 @@ func getLegacyVolumeName(mountPath string) (string, error) {
 func (ns *NodeServer) mountVolumeToStagePath(ctx context.Context, req *csi.NodeStageVolumeRequest, stagingPath, devicePath string) error {
 	// Publish Path
 	fsType := req.GetVolumeCapability().GetMount().GetFsType()
-	diskMounter := &mount.SafeFormatAndMount{Interface: ns.mounter, Exec: mount.NewOsExec()}
+
+	// Use 'ext4' as the default
+	if fsType == "" {
```

Comment:
note: `ext4` is defaulted in external-provisioner if no fstype is specified in SC